src/utils/util.ts

```diff
@@ -17,7 +17,17 @@ export const copyElementValue = (component: any) => {
 }
 
 export const shannonToCkb = (value: number) => {
-  return value / 10 ** 8
+  const num = value / 10 ** 8
+  if (num < 10 ** -8) {
```

Comment:
There might be a precision problem with `10**-8`, use `big-number` is a better choice.

Reply:
OK

Reply:
@Keith-CY  Fixed. https://github.com/nervosnetwork/ckb-explorer-frontend/pull/307/commits/0212d2ab8287524b8c08915fcbc9c60473466ee1

And I will refactor all capacity, reward and fee value with BigNumber in another PR later.